scalafmt-core/shared/src/main/scala/org/scalafmt/Error.scala

```diff
@@ -80,7 +80,7 @@ object Error {
   ) extends Error({
         val tok = LoggerOps.log2(ft)
         val line = ft.left.pos.endLine
-        s"Search state exploded on '$tok', line $line"
+        s"Search state exploded on '$tok', line $line.\nConsider increasing runner.optimizer.maxVisitsPerToken (default 10000) or even runner.maxStateVisits (default 1000000).\nSee https://scalameta.org/scalafmt/docs/configuration.html#search-state-exploded. "
```

Comment:
please include only the doc link, there's no need to repeat the recommendation.

Reply:
After following the link, I had to look around the documentation to find the default values of these properties in order to know to what value to increase them. I think it's more useful for a user that has to deal with this exception to provide the recommendation and the default values right away in the error message.

If it's all the same to you, I sent another commit removing the link in order not to have duplicate information.